File: scripts/java/NetflixOSS-pocs/karyon-poc/src/main/java/com/github/diegopacheco/sandbox/java/netflixoss/karyon/poc/jersey/MainRunner.java
package com.github.diegopacheco.sandbox.java.netflixoss.karyon.poc.jersey;

public class MainRunner {
	public static void main(String[] args) {
		System.setProperty("java.awt.headless","true");
		System.setProperty("archaius.deployment.environment","dev");
		netflix.karyon.KaryonRunner.main(new String[]{KaryonJerseyServerApp.class.getCanonicalName()});
	}
}

File: scripts/scala/scala-karyon-sbt-native-packager/src/main/scala/com/github/diegopacheco/sandbox/scala/netflix/karyon/MainRunner.scala
package com.github.diegopacheco.sandbox.scala.netflix.karyon

object MainRunner extends App {
    System.setProperty("java.awt.headless","true")
		System.setProperty("archaius.deployment.environment","dev")
		val s:String = classOf[KaryonJerseyServerApp].getCanonicalName()
		netflix.karyon.KaryonRunner.main(Array(s))
}